cmd/semantic-release/main.go

```diff
@@ -200,7 +200,7 @@ func cliHandler(cmd *cobra.Command, args []string) {
 	}
 
 	logger.Println("getting commits...")
-	rawCommits, err := prov.GetCommits(release.SHA, currentSha)
+	rawCommits, err := prov.GetCommits(repoInfo.DefaultBranch, currentBranch)
```

Comment:
This may create some weird corner case bugs with CI re-runs or overlapping builds when there's later commits on the branch.